packages/oui-status/_variables.less

```diff
@@ -0,0 +1,24 @@
+// Status base
+@oui-status-font-size: rem-calc(16);
```

Comment:
New rules (again)! :tada:
Use directly the color from `_colors` in the component.
Set directly the padding without making variable except if it's really useful.
Globally, prefer using the global variables instead of set new (useless) variables for the component.